code/controllers/subsystem/ticker.dm

```diff
@@ -662,17 +662,23 @@ SUBSYSTEM_DEF(ticker)
 	save_admin_data()
 	update_everything_flag_in_db()
 	if(!round_end_sound)
-		round_end_sound = pick(\
-		'sound/roundend/newroundsexy.ogg',
-		'sound/roundend/apcdestroyed.ogg',
-		'sound/roundend/bangindonk.ogg',
-		'sound/roundend/leavingtg.ogg',
-		'sound/roundend/its_only_game.ogg',
-		'sound/roundend/yeehaw.ogg',
-		'sound/roundend/disappointed.ogg',
-		'sound/roundend/scrunglartiy.ogg',
-		'sound/roundend/petersondisappointed.ogg'\
-		)
+		// FULP-ONLY STARTS - Adds a pointer to the new modular pick_round_end_sound() proc from Fulp's sound.dm
+		// Original code left in comment
+
+		round_end_sound = pick_round_end_sound()
+
+		// round_end_sound = pick(\
```

Comment:
couldnt you comment this out with /* */ instead, so TG conflicts wouldnt actually conflict as much?

Reply:
Also it would likely fix the linter issue you have, instead of keep blaming the master being bad.